src/test/user.spec.js

```diff
@@ -37,7 +42,18 @@ describe('Test user login, signup and account verification', () => {
         });
     });
 
-    it('it should throw an error when user with an unverified account wants to signup', done => {
+    it('should mock a sendgrid api request', async () => {
+      const secret = process.env.SENDGRID_API_KEY;
+      sgMail.setApiKey(secret);
+
+      nock('https://api.sendgrid.com')
+        .post('/api/mail.send.json')
+        .reply(200, {
+          message: 'success'
+        });
+    });
+
+    it('should throw an error when user with an unverified account wants to signup', done => {
       const newUser = {
```

Comment:
I don't know the reason there are two it statement in these lines of code please could you visit this?